src/script/view_model/content/PreferencesAccountViewModel.ts

```diff
@@ -45,19 +47,55 @@ import {AvailabilityContextMenu} from '../../ui/AvailabilityContextMenu';
 import {MotionDuration} from '../../motion/MotionDuration';
 import {EventName} from '../../tracking/EventName';
 import {ContentViewModel} from '../ContentViewModel';
+import {Logger} from '@wireapp/commons';
+import {getLogger} from 'Util/Logger';
 
 import 'Components/availabilityState';
 import {isAppLockEnabled} from './AppLockViewModel';
 import {loadValue} from 'Util/StorageUtil';
 import {StorageKey} from '../../storage';
 import {UserError} from '../../error/UserError';
 import {HistoryExportViewModel} from './HistoryExportViewModel';
+import {ClientRepository} from '../../client/ClientRepository';
+import {ConversationRepository} from '../../conversation/ConversationRepository';
+import {TeamRepository} from '../../team/TeamRepository';
+import {AccentColorID} from '@wireapp/commons/dist/commonjs/util/AccentColor';
+import {TeamEntity} from '../../team/TeamEntity';
+
+export class PreferencesAccountViewModel {
+  logger: Logger;
+  fileExtension: string;
+  isDesktop: boolean;
+  brandName: string;
+  isActivatedAccount: ko.PureComputed<boolean>;
+  selfUser: ko.Observable<User>;
+  name: ko.PureComputed<string>;
+  availability: ko.PureComputed<Availability.Type>;
+  availabilityLabel: ko.PureComputed<string>;
+  username: ko.PureComputed<string>;
+  enteredUsername: ko.Observable<string>;
+  submittedUsername: ko.Observable<string>;
+  usernameState: ko.Observable<string>;
+  richProfileFields: ko.Observable<any[]>;
+  nameSaved: ko.Observable<boolean>;
+  usernameSaved: ko.Observable<boolean>;
+  isTeam: ko.PureComputed<boolean>;
+  team: ko.Observable<TeamEntity>;
+  teamName: ko.PureComputed<string>;
+  optionPrivacy: ko.Observable<boolean>;
+  optionReadReceipts: ko.Observable<any>;
```

Comment:
Can we type this?

Reply:
thanks, done in https://github.com/wireapp/wire-webapp/pull/9094/commits/812138977097146701e886804188567abd5cced5